docs/concepts/included_models.rst

```diff
@@ -0,0 +1,286 @@
+This is useful for representing relative transformations between a series of small molecules without invoking the additional complexity of an :ref:`alchemicalnetwork`.
+
+.. note::
+    The :class:`.Component` is an *extensible point* of the library,
+    and is intended to be subclassed to enable new applications.
+    For details on how to create your own :class:`.Component` classes, see :ref:`howto-component`.
+
+
+.. _chemicalsystem:
+
+``ChemicalSystem``
+------------------
+
+A :class:`.ChemicalSystem` represents a complete system of molecules,
+and is often composed of multiple :ref:`Components <component>`.
+
+These are most often used as nodes of an :ref:`alchemicalnetwork`, with pairs of ``ChemicalSystem``\s connected by :ref:`Transformations <transformation>`.
+Because a ``ChemicalSystem`` functions as a kind of container of :ref:`Components <component>`, more than one ``ChemicalSystem`` can feature the same ``Component``\s.
+This allows even very large ``AlchemicalNetwork``\s to be relatively small in memory, as only a few large ``Component``\s like :class:`.ProteinComponent`\s may be shared among hundreds of ``ChemicalSystem``\s.
+
+When used as inputs to a ``Transformation``, ``ChemicalSystem``\s represent the set of ``Component``\s for which a free energy difference will be estimated.
+Alchemical methods performing free energy perturbation (FEP) between the two ``ChemicalSystem``\s of a ``Transformation`` will simulate these ``Component``\s using some sampling approach, obtaining enough information to derive a free energy difference estimate.
+
+
+.. _transformation:
+
+``Transformation``
+------------------
+
+A :class:`.Transformation` represents an alchemical transformation between two :ref:`ChemicalSystems <chemicalsystem>`.
+
+``Transformation`` objects are often used as the edges of an :ref:`alchemicalnetwork`.
+In addition to referencing the ``ChemicalSystem``\s it spans,
+a ``Transformation`` also includes the :ref:`protocol` used to actually perform the alchemical transformation,
+as well as an :ref:`componentmapping` specifying what portions of the :ref:`Components <component>` are being transformed across the ``ChemicalSystem``\s.
+
+A ``Transformation`` functions as a container for all the information needed to obtain an estimate of the free energy difference between its two ``ChemicalSystem``\s.
+
+
+.. _nontransformation:
+
+``NonTransformation``
+---------------------
+
+A :class:`.NonTransformation` represents non-alchemical sampling of a single :ref:`ChemicalSystem <chemicalsystem>`.
+
+In the context of an :ref:`alchemicalnetwork`, a ``NonTransformation`` is effectively a self-loop, featuring the same ``ChemicalSystem`` at either end.
+Similar to a :ref:`Transformation <transformation>`, it features a :ref:`protocol` used to perform sampling on its ``ChemicalSystem``, but does not feature a :ref:`componentmapping` since none is required for this.
```

Comment:
```suggestion
Similar to a :ref:`Transformation <transformation>`, it features a :ref:`protocol` used to perform sampling on its ``ChemicalSystem``, but does not feature a :ref:`componentmapping`, since there is no second ``ChemicalSystem``.
```
it's not just not required, it cannot/should not be defined.